lib/ansible/modules/system/cronvar.py

```diff
@@ -126,6 +126,7 @@ def __init__(self, module, user=None, cron_file=None):
         self.user = user
         self.lines = None
         self.wordchars = ''.join(chr(x) for x in range(128) if chr(x) not in ('=', "'", '"',))
+        self.cron_cmd = self.module.get_bin_path('cronvar', required=True)
```

Comment:
This seems to refer a non-existent binary `cronvar`, instead of previously used `crontab`. With this change the `cronvar` ansible module fails:
```console
{"changed": false, "msg": "Failed to find required executable cronvar in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin"}
```